perfkitbenchmarker/vm_util.py

```diff
@@ -525,25 +525,30 @@ def IssueCommand(cmd, force_info_log=False, suppress_warning=False,
   logging.info('Running: %s', full_cmd)
 
   shell_value = RunningOnWindows()
-  process = subprocess.Popen(cmd, env=env, shell=shell_value,
-                             stdin=subprocess.PIPE, stdout=subprocess.PIPE,
-                             stderr=subprocess.PIPE)
+  with tempfile.TemporaryFile() as tf_stderr:
+    with tempfile.TemporaryFile() as tf_stdout:
```

Comment:
I'll shorten variable names. I think `tf_out` and `tf_err` should still be understandable.